### PR TITLE
[NETPROD-3585] Added name filter in doctl for certificates

### PR DESCRIPTION
**Pull request by guptado**
[NETPROD-3585](https://do-internal.atlassian.net/browse/NETPROD-3585)

[NETPROD-3585]: https://do-internal.atlassian.net/browse/NETPROD-3585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by apinonformoso — COMMENTED** [review]
Can the changes be tested locally apart from the unit tests?

**Comment by guptado** [comment]
> Can the changes be tested locally apart from the unit tests?

Yes. We can do go install & try it. I was not able to do it on my machine due to multiple set up issues. But this pattern & code is being used everywhere so it gives confidence

**Review by apinonformoso — DISMISSED** [review]

File: commands/certificates.go [hunk 1]
```diff
@@ -78,6 +78,8 @@ To upload a custom certificate, you need to provide a certificate name, the path
 	cmdCertificateList := CmdBuilder(cmd, RunCertificateList, "list", "Retrieve list of the account's stored certificates", `This command retrieves a list of all certificates associated with the account. The following details are shown for each certificate:`+certDetails, Writer,
 		aliasOpt("ls"), displayerType(&displayers.Certificate{}))
 	cmdCertificateList.Example = `The following example retrieves a list of all certificates associated with your account and uses the ` + "`" + `--format` + "`" + ` flag return only the IDs, names, and the domains associated with each ticket: doctl compute certificate list --format ID,Name,DNSNames`
+	AddStringFlag(cmdCertificateList, doctl.ArgCertificateName, "", "",
+		"name of the certificate to be listed")
```
Comment (andrewsomething) [inline commands/certificates.go:82]:
```suggestion
		"Filter certificates by the specified name")
```

**Comment by andrewsomething** [comment]
@guptado, @apinonformoso - You should be able to build doctl locally by running `make build`. Please let us know what kinds of issues you were running into so we can improve the contributor experience.

**Comment by guptado** [comment]
> @guptado, @apinonformoso - You should be able to build doctl locally by running `make build`. Please let us know what kinds of issues you were running into so we can improve the contributor experience.

Thanks! It seems to be working fine. @apinonformoso below is the output the commands

`./builds/doctl compute certificate list --name bindiya-certificate                 
ID                                      Name                   DNS Names                              SHA-1 Fingerprint                           Expiration Date         Created At              Type            State
33c30d70-908a-4ed2-9c4f-6f2544071c11    bindiya-certificate    fertileearth.in,cdn.fertileearth.in    ae4af8aaf7f4011a383da3997b8b06c1fb748529    2024-05-05T17:30:04Z    2024-02-05T18:30:02Z    lets_encrypt    verified
`

**Pull request merged**